docs/pages/content/schemas.tsx

```diff
@@ -11,8 +11,8 @@ const SchemasPage: React.FunctionComponent = () => {
       <h1>Schema Groups</h1>
 
       <p>
-        This is the schemas exported by nimbus-shared. The source of these
-        schemas is Typescript on <a href={githubUrl}>on Github</a>.
+        This is the schemas exported by nimbus-shared. The source of these schemas is Typescript on{" "}
+        <a href={githubUrl}>on Github</a>.
```

Comment:
uber nit: If I'm reading this correctly, you'll have a double "on on" here.

Reply:
You are reading this correctly. I'll fix that.

Reply:
Actually, where is this page on GitHub pages? Based on the filename/path, I assumed it'd be the Schemas page at https://mozilla.github.io/nimbus-shared/dev/schemas but I'm not seeing this content.

Or is GitHub pages not updated on each PR merge?

Reply:
The deploy job has a problem that I need to fix. It should auto update, but it isn't.

Reply:
Ok, bug fixed, deploys are working again.

Also, this page is at https://mozilla.github.io/nimbus-shared/content/schemas. The paths in the repo (`docs/pages/content/schemas.tsx`) get mapped pretty directly to the HTTP space: `content/schemas.html`, and then there is a 404-router that redirects `content/schemas` to that.